toke.c

```diff
@@ -7349,7 +7349,7 @@ yyl_fake_eof(pTHX_ U32 fake_eof, bool bof, char *s)
         if (PL_parser->in_pod) {
             /* Incest with pod. */
             if (    memBEGINPs(s, (STRLEN) (PL_bufend - s), "=cut")
-                && !isALPHA(s[4]))
+                && !isWORDCHAR_A(s[4]))
```

Comment:
This should probably instead be `!isIDCONT_A` for Unicode-aware use.  The two macros are equivalent in the ASCII range, but at some point we may want to expand beyond that range, and the proper semantics are IDCONT

Reply:
Done.